src/ReactiveUI/Platforms/windows-common/ReactivePage.cs

```diff
@@ -100,6 +104,40 @@ class ReactivePage<TViewModel> :
                 typeof(ReactivePage<TViewModel>),
                 new PropertyMetadata(null));
 
+#if HAS_UNO
+#if ANDROID
+        /// <summary>
+        /// Initializes a new instance of the <see cref="ReactivePage{TViewModel}"/> class.
+        /// Native constructor, do not use explicitly.
+        /// </summary>
+        /// <remarks>
+        /// Used by the Xamarin Runtime to materialize native
+        /// objects that may have been collected in the managed world.
+        /// </remarks>
+        /// <param name="javaReference">javaReference.</param>
```

Comment:
Put a valid comment in the Params

Reply:
TBH, I don't know myself.
I tried to look up the docs from [here](https://github.com/unoplatform/uno/blob/master/src/Uno.UI/UI/Xaml/Controls/Page/NativePage.Android.cs#L14) but it says nothing.